src/package.xml

```diff
@@ -1,21 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
-    <types>
-        <members>Insert_Student_Details</members>
-        <members>StudentQuery</members>
-        <name>ApexClass</name>
-    </types>
     <types>
         <members>Student__c</members>
+        <members>Pan_Card__c</members>
+        <members>Student__c</members>
+        <members>StudentSubjects__c</members>
+        <members>Subject__c</members> 
+        <members>Teacher__c</members> 
+        <members>Year_of_Class__c</members>
+        <members>College__c</members>
         <name>CustomObject</name>
     </types>
     <types>
         <members>Student__c</members>
+        <members>Pan_Card__c</members>
+        <members>Student__c</members>
+        <members>StudentSubjects__c</members>
+        <members>Subject__c</members> 
+        <members>Teacher__c</members> 
+        <members>Year_of_Class__c</members>
+        <members>College__c</members>
         <name>CustomTab</name>
     </types>
     <types>
-        <members>Student__c-Registrar Layout</members>
-        <members>Student__c-Teacher Layout</members>
+        <members>Pan_Card__c-Pan Card Layout</members>
```

Comment:
Why were these layouts created? these are not requested in the assignment for now. Remove them all.